test.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+export OPENAI_API_KEY=`cat .api_key`
```

Comment:
would be nice to have an option for external export:
```suggestion
export OPENAI_API_KEY=${OPENAI_API_KEY:-$(cat .api_key)}
```